src/resolver/range-request.ts

```diff
@@ -39,6 +39,14 @@ export async function requestRanges(
     return await resolveIndividualPromises(url, ranges);
   }
 
+  for (const { start, end } of ranges) {
+    if (end - start === 0) {
```

Comment:
```suggestion
    if (end - start <= 0) {
```

Negative would also be bad.